### PR TITLE
allow setting of network card model

### PR DESCRIPTION
**Pull request by abbbi**
see #521 
this time the testacc shall work as resourceLibvirtDomainRead has been updated accordingly too.

### CODE REVIEW

**Comment by MalloZup** [comment]
thx @abbbi  for  this.  I will have a look once i have some free time! thank you for pr

**Comment by abbbi** [comment]
Updated the PR to also add a test in TestAccLibvirtDomain_NetworkInterface

File: website/docs/r/domain.html.markdown [hunk 1]
```diff
@@ -353,6 +353,7 @@ When using a virtual network, users can specify:
   [network resource](/website/docs/r/network.markdown) managed by the
   Terraform libvirt provider.
 * `mac` - (Optional) The specific MAC address to use for this interface.
+* `model` - Set the model of emulated network interface card, defaults to `virtio`.
```
Comment (inercia) [inline website/docs/r/domain.html.markdown]:
```suggestion
* `model` - (Optional) Set the model of emulated network interface card, defaults to `virtio`.
```

**Comment by MalloZup** [comment]
Closing as  commented.@abbbi thx if you want to help on other topics feel free to grab some open issue 🎸🎶

**Pull request closed without merging**